allen-cell-animated/simularium-viewer
# Feature/error levels

Problem
=======
closes #219 
closes #201 

Solution
========
Added a level to front end error
Made all emitted errors FrontEndErrors
Left htmlData as a field because of how it's being used on the front end.

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires updated or new tests

Steps to Verify:
----------------
I pushed my local build to the website repo if you want to test out these changes
2. View this [diff](https://github.com/allen-cell-animated/simularium-website/compare/local-viewer-install?expand=1)
3. Copy changes locally, plus install your viewer locally
4. Drag and drop the actin file without any geometry. 
5. You should never see the panel flash into error state, it goes from loading to success.
